emails/management/commands/process_delayed_emails_from_sqs.py

```diff
@@ -69,8 +69,12 @@ def handle(self, *args, **options):
         )
         while len(messages) > 0:
             for message in messages:
-                _verify_and_run_sns_inbound_on_message(message)
-
+                try:
+                    _verify_and_run_sns_inbound_on_message(message)
+                except:
+                    exc_type, _, _ = sys.exc_info()
+                    logger.exception(f'dlq_processing_error_{exc_type}')
+                    message.delete()
```

Comment:
I wonder if it would be best to use `try...except...finally` here? So remove the 2 calls to `message.delete` that are inside `_verify_and_run_sns_inbound_on_message`, move this call into a `finally:` block?

Reply:
Sounds good. That will make it more clear that deleting will happen irregardless.

Reply:
Done on 54ca46b